### ISSUE TITLE
Windows 10 -- path issue?

### ISSUE THREAD

**Issue by krzysztofik** [protz/LatexIt#15]
I went through the Troubleshooting and everything works fine there. However, after running LatexIt in my test email message I got the following report:

------------------------------------------
LatexIt! run report...

*** Found expression $x=1$

$PATH is C:\Program Files (x86)\Mozilla Thunderbird;C:\Program Files\ImageMagick-6.9.3-Q16;C:\ProgramData\Oracle\Java\javapath;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\Intel\iCLS Client\;C:\Program Files\Common Files\Microsoft Shared\Windows Live;C:\Program Files (x86)\Common Files\Microsoft Shared\Windows Live;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Windows Live\Shared;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files (x86)\Intel\OpenCL SDK\2.0\bin\x86;C:\Program Files (x86)\Intel\OpenCL SDK\2.0\bin\x64;C:\Program Files (x86)\MiKTeX 2.9\miktex\bin\;C:\Program Files\Samsung\AllShare Framework DMS\1.3.06\;C:\Program Files\Samsung\AllShare Framework DMS\1.3.06\64bit\;C:\strawberry\c\bin;C:\strawberry\perl\site\bin;C:\strawberry\perl\bin;C:\Program Files (x86)\gnuplot\bin;C:\Program Files (x86)\QuickTime\QTSystem\;C:\Program Files\OpenVPN\bin;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Calibre2\;C:\Program Files (x86)\Skype\Phone\

*** Generating LaTeX expression:
\documentclass{article}
\usepackage[utf8x]{inputenc}
\pagestyle{empty}
\begin{document}
$x=1$ %this is where your LaTeX expression goes
\end{document}

I ran C:\Program Files (x86)\MiKTeX 2.9\miktex\bin\latex.exe -output-directory=C:\Users\KRZYSZ~1\AppData\Local\Temp -interaction=batchmode C:\Users\KRZYSZ~1\AppData\Local\Temp\tblatex-1.tex error code 1
LaTeX process returned 1
Proceeding anyway...
Severe error. Missing package?
We left the .tex file there: C:\Users\KRZYSZ~1\AppData\Local\Temp\tblatex-1.tex, try to run latex on it by yourself...
--> Failed, not inserting
------------------------------------------

I tried adding quotation marks to the paths to my executable latex, dvips and convert, but it also gives errors: 

------------------------------------------
This path is malformed: "C:\Program Files (x86)\MiKTeX 2.9\miktex\bin\latex.exe".
Possible reasons include: you didn't setup the paths properly in the addon's options.
Wrong path for latex bin. Please set the right path in the options dialog first.
--> Failed, not inserting
------------------------------------------

Any hints?

**Comment by protz** [comment]
Hmm I think there's been issues with TexLive 2014. If that's what you're 
using, use TexLive 2015.

**Comment by krzysztofik** [comment]
I'm using MikTeX (my package manager says 2.9.5700).

**Comment by protz** [comment]
Have you tried following the instructions in troubleshooting? That is, 
running exactly the same command as LatexIt...

**Comment by protz** [comment]
Also, two things:
- you don't need quotation marks
- I see latexexe instead of latex.exe; is that normal?

**Comment by krzysztofik** [comment]
Yes, I followed the troubleshooting instructions and all worked fine for me. 
> I see latexexe instead of latex.exe; is that normal?
Not sure what you mean here. My paths to the .exe files are the following:
C:\Program Files (x86)\MiKTeX 2.9\miktex\bin\latex.exe
C:\Program Files (x86)\MiKTeX 2.9\miktex\bin\dvips.exe
C:\Program Files\ImageMagick-6.9.3-Q16\convert.exe

**Comment by krzysztofik** [comment]
Also, I use the default LaTeX template.

**Comment by msprotz** [comment]
So if you run `C:\Program Files (x86)\MiKTeX 2.9\miktex\bin\latex.exe -output-directory=C:\Users\KRZYSZ~1\AppData\Local\Temp -interaction=batchmode C:\Users\KRZYSZ~1\AppData\Local\Temp\tblatex-1.tex` in a cmd prompt, it works?

**Comment by krzysztofik** [comment]
No. Here's the log:

C:\Users\Krzysztof\Desktop>C:\Program Files (x86)\MiKTeX 2.9\miktex\bin\latex.exe -output-directory=C:\Users\KRZYSZ~1\AppData\Local\Temp -interaction=batchmode C:\Users\KRZYSZ~1\AppData\Local\Temp\tblatex-1.tex
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.

I added quotation marks to the initial path and I got:

C:\Users\Krzysztof\Desktop>"C:\Program Files (x86)\MiKTeX 2.9\miktex\bin\latex.exe" -output-directory=C:\Users\KRZYSZ~1\AppData\Local\Temp -interaction=batchmode C:\Users\KRZYSZ~1\AppData\Local\Temp\tblatex-1.tex
This is pdfTeX, Version 3.14159265-2.6-1.40.16 (MiKTeX 2.9)
entering extended mode

C:\Users\Krzysztof\Desktop>

(that's it)
